packages/core/src/sass/app/_common/layout/app-layouts.scss

```diff
@@ -185,11 +194,7 @@
         &--is-hidden {
             margin-top: 0;
         }
-        .ciq-menu {
```

Comment:
Sorry for silly question, but we need to delete this innocent style, right?

Reply:
Actually i just moved the style above in `@include mobile-screen`, but maybe i can remove it

Reply:
I mean I'm not sure, why you removed .ciq-menu )

Reply:
Ah, okay, got it